CommonTools/UtilAlgos/interface/EventSelectorAdapter.h

```diff
@@ -17,6 +17,14 @@
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "HLTrigger/HLTcore/interface/defaultModuleLabel.h"
+
+template <typename T>
+struct FillDescriptionTraits {
+  static std::string moduleLabel() { return defaultModuleLabel<T>(); }
+};
```

Comment:
If we take this route, obviously this definition must go to some more "central" header (and the class template to be renamed and/or put in a namespace).

Reply:
I think that this implementation is not very useful because it requires knowledge of the mangled name, while in reality it's the registered module plugin name that is the most relevant default.